pkg/ha/ha_tracker.go

```diff
@@ -212,6 +218,12 @@ func NewHATracker(cfg HATrackerConfig, limits HATrackerLimits, trackerStatusConf
 		}
 		t.client = client
 	}
+	go func() {
+		for {
+			time.Sleep(10 * time.Second)
```

Comment:
Let's do 30s interval and make the interval a const

Reply:
for sure! :)